File: test/fixtures/output/python/requests/full.py
import requests

url = "http://mockbin.com/har"

querystring = {'name':'foo','value':'bar','name':'foo','value':'baz','name':'baz','value':'abc'}

payload = "foo=bar"
headers = {
    'cookie': "foo=bar; bar=baz",
    'accept': "application/json",
    'content-type': "application/x-www-form-urlencoded"
    }

response = requests.request("POST", url, data=payload, headers=headers, params=querystring)

print(response.text)

File: test/fixtures/output/node/axios/full.js
const axios = require("axios").default;

const { URLSearchParams } = require('url');
const encodedParams = new URLSearchParams();

encodedParams.set('foo', 'bar');

const options = {
  method: 'POST',
  url: 'http://mockbin.com/har',
  params: {foo: ['bar', 'baz'], baz: 'abc', key: 'value'},
  headers: {
    cookie: 'foo=bar; bar=baz',
    accept: 'application/json',
    'content-type': 'application/x-www-form-urlencoded'
  },
  data: encodedParams
};

axios.request(options).then(function (response) {
  console.log(response.data);
}).catch(function (error) {
  console.error(error);
});
